node/pkg/watchers/evm/watcher.go

```diff
@@ -316,6 +316,26 @@ func (w *Watcher) Run(ctx context.Context) error {
 			p2p.DefaultRegistry.AddErrorCount(w.chainID, 1)
 			return fmt.Errorf("failed to create polygon connector: %w", err)
 		}
+	} else if w.chainID == vaa.ChainIDBase && !w.unsafeDevMode {
+		baseConnector, err := connectors.NewEthereumConnector(timeout, w.networkName, w.url, w.contract, logger)
+		if err != nil {
+			ethConnectionErrors.WithLabelValues(w.networkName, "dial_error").Inc()
+			p2p.DefaultRegistry.AddErrorCount(w.chainID, 1)
+			return fmt.Errorf("dialing eth client failed: %w", err)
+		}
+		pollConnector, err := connectors.NewBlockPollConnector(ctx, baseConnector, finalizers.NewDefaultFinalizer(), 250*time.Millisecond, true, true)
```

Comment:
Why is the pollconnector config different from arbitrum?

Reply:
Base is OP stack (Optimism) which is the Bedrock release, and so it support `safe` and `finalized` tags. I believe that's why it matches Eth in this way.

Reply:
That is correct. Discussed this with @hendrikhofstadt offline. I think we are good on this.